### DISCUSSION TITLE
text-align property

### DISCUSSION THREAD

**Discussion by vikas-growisto** [fastn-stack/fastn#588] [Q&A]
```
-- ftd.text: FifthTry
width.fixed.px: 200
text-align: left
```
The property is not working can you provide an example that is working with the property.
text-align: right is also not working.
Only text-align: center is working.

**Comment by Arpita-Jaiswal** [comment]
We support `start`, `center`, `end` and `justify` values. Please checkout [ftd cheatsheet](https://github.com/ftd-lang/ftd/blob/main/Cheatsheet.md).
